components/Style/Button.styled.js

```diff
@@ -0,0 +1,116 @@
+import styled from "styled-components";
+import { css } from "styled-components";
+
+const StyledButton = styled.button`
+  cursor: pointer;
+
+  ${({ variant }) =>
+    variant === "submit" &&
+    css`
+      background-color: var(--tertiary-color);
+      width: 176px;
+      height: 35px;
+      border-radius: var(--border-radius);
+      border: none;
+      color: var(--primary-color);
+      font-size: 16px;
+      align-self: center;
+    `}
+
+  ${({ variant }) =>
+    variant === "contact" &&
+    css`
+      background-color: var(--tertiary-color);
+      width: 176px;
+      height: 35px;
+      border-radius: var(--border-radius);
+      border: none;
+      color: var(--primary-color);
+      font-size: 16px;
+      display: flex;
+      align-items: center;
+      justify-content: space-evenly;
+      margin: 0 auto;
+      padding: 0 20px;
+    `}
```

Comment:
```suggestion
  ${({ variant }) =>
    variant === "contact" &&
    css`
      background-color: var(--tertiary-color);
      width: 175px;
      height: 35px;
      border-radius: var(--border-radius);
      border: none;
      color: var(--primary-color);
      font-size: 1rem;
      display: flex;
      align-items: center;
      justify-content: space-evenly;
      margin: 0 auto;
      padding: 0 20px;
    `}
```

Reply:
thank you a lot Jessica. Very very good feedback